src/crate/qa/tests.py

```diff
@@ -181,7 +184,8 @@ def _unicast_hosts(self, num, transport_port=4300):
             for x in range(num)
         ])
 
-    def _new_cluster(self, version, num_nodes, settings=None, env=None):
+    def _new_cluster(self, version, num_nodes, data_paths=None, settings=None, env=None):
```

Comment:
there is also `_new_heterogeneous_cluster` - but I didn't change it as it's not used (usages was removed in https://github.com/crate/crate-qa/commit/10bc51176ff71e6015d9bdc91b56781c4fe5c7a4 and https://github.com/crate/crate-qa/commit/66b2b68879fb1e323b94fe7da10e5a90942be43d). 
Should I remove it in a separate commit?